tests/Hedgehog.CSharp.Tests/NameTests.cs

```diff
@@ -9,9 +9,9 @@ namespace Hedgehog.CSharp.Tests
     public class NameTests
     {
         private static Type[] _publicApiTypes =
-            { typeof(Gen)
-            , typeof(Range)
-            , typeof(Property)
+            { typeof(Hedgehog.CSharp.Gen)
+            , typeof(Hedgehog.CSharp.Range)
+            , typeof(Hedgehog.CSharp.Property)
             };
 
         public static IEnumerable<object[]> AllPublicMembers()
```

Comment:
This method contains
```csharp
// Ignore static inline methods
// by convention we mark them as starting with '`'
if (mi != null && mi.Name.StartsWith("`"))
```

Are there still function names that start with that character?

Reply:
Removing that statement doesn't cause any existing tests to fail, I think it can be discarded.

Reply:
I will look into this tomorrow

Reply:
I have tested now as well.  I also think it can be removed.

The purpose of this test is to ensure that all the methods exposed for the purpose of C# start with a capital letter as is idiomatic for C#.  That special character was put at the beginning of inlined functions by convention. which are functions that don't have an equivalent for C#.  Now that you have separated the C# API, there are no such functions.

Reply:
I've left this in, and put the `CompiledName` attributes back in for a selection of the functions identified by @moodmosaic